test/e2e/provisioning/helper.go

```diff
@@ -155,13 +155,13 @@ func testScenario(t *testing.T, testCase scenario, cloudProvider string, testPar
 func buildScenarios() []scenario {
 	var all []scenario
 	for _, version := range versions {
-		for _, os := range operatingSystems {
+		for _, ops := range operatingSystems {
```

Comment:
Why did you rename this?

Reply:
because a package called `os` is called:
https://github.com/kubermatic/machine-controller/pull/595/files/72ab586e9bcce062034ea5ceb3fdb7b3a2d24f16#diff-4b71c77de88f8c32fb99c651cc08ab1fL21

Reply:
That link doesn't work for me. If the thing shadows something, can you just rename the var to `operatingSystem`? The current name could also be random-generated

Reply:
well in go, generated vars are actually started with `_`, and its a derived variable from the slice `operatingSystems`

Reply:
What exactly is generated about this var? This name doesn't reflect at all what it is, please write readable code.

Reply:
There is nothing generated about it, you said `The current name could also be random-generated`, so I told you generated vars in go usually starts with  `_`, and to be honest the code is really readable, the slice is called `operatingSystem`, and `ops` is a part of that slice, what is not reliable about  ???